pkgs/development/python-modules/piexif/default.nix

```diff
@@ -0,0 +1,24 @@
+{lib, buildPythonPackage, fetchurl, pillow}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "piexif";
+  version = "1.0.12";
+
+  # pillow needed for unit tests
+  buildInputs = [ pillow ];
+
+  # No .tar.gz source available at PyPI, only .zip source, so need to use
+  # fetchurl because fetchPypi doesn't support .zip.
+  src = fetchurl {
+    url = "mirror://pypi/packages/source/p/${pname}/${name}.zip";
```

Comment:
I made that change. Strange that I didn't get any download failures with that previous url.